locales/en/translation.json

```diff
@@ -495,16 +500,18 @@
   "walkthroughPlanCreateEpic": "Epics represent larger changes to the Project. You can invite multiple Collaborators to your Epic and assign different people as Developers and Testers for each Task.",
   "walkthroughPlanCreateTask": "To get started contributing to this Project, create a Task. Tasks represent small changes to this Project; each one has a Developer and a Tester. Tasks are equivalent to GitHub branches.",
   "walkthroughPlanListEpics": "Select the Epics tab to see a list of all the Epics for this Project. Each Epic is a group of related Tasks.",
-  "walkthroughPlanListTasks": "Select the Tasks tab to see a list of all the work being done on this Project and who is doing it. Tasks represent small changes to the Project and may be grouped with other Tasks in an Epic.",
+  "walkthroughPlanListTasks": "Select the Tasks tab to see a list of all the work being done on this Project and who is doing it. Tasks represent small changes to the Project, and may be part of an Epic.",
+  "walkthroughPlayCreateScratchOrg": "Scratch Orgs are a temporary place for you to view the work on this Project. You can use Scratch Orgs to play with changes to the Project without affecting the Project.",
   "walkthroughPlayListEpics": "Select the Epics tab to see a list of all the Epics for this Project. Each Epic is a group of related Tasks. Select an Epic and create an Epic Scratch Org to view the work on that Epic.",
-  "walkthroughPlayListTasks": "Select the Tasks tab to see a list of all the work being done on this Project and who is doing it. Tasks represent small changes to the Project and may be grouped with other Tasks in an Epic. Select a Task and create a Task Scratch Org to view the work on that Task.",
-  "walkthroughPlayMetechoName": "Metecho makes it easier for you to view, test, and contribute to Salesforce Projects without learning GitHub.<1></1><2></2><3>Pronunciation</3>: “Met” rhymes with “Bet.” “Echo” as in the reflection of sound waves.<5></5><6>Definition</6>: “Share or participate in.”",
-  "walkthroughPlayScratchOrg": "Scratch Orgs are a temporary place for you to view the work on this Project. You can use Scratch Orgs to play with changes to the Project without affecting the Project.",
+  "walkthroughPlayListTasks": "Select the Tasks tab to see a list of all the work being done on this Project and who is doing it. Tasks represent small changes to the Project, and may be part of an Epic. Select a Task and create a Task Scratch Org to view the work on that Task.",
+  "walkthroughPlayMetechoName": "Metecho makes it easier for you to view, test, and contribute to Salesforce Projects without learning GitHub.<1></1><2></2><3>Pronunciation</3>: “Met” rhymes with “bet.” “Echo” as in the reflection of sound waves.<5></5><6>Definition</6>: To share or participate in.",
+  "walkthroughPlayScratchOrg": "This is a sample Scratch Org for this Project. Select “View Org” to see the work on this Project. Your Org will expire after 30 days.",
```

Comment:
I was tempted to click "View Org" during the walkthrough. Clicking anything in that highlighlighted space closes the walkthrough. Should a click like this do nothing and keep the tour open?

Reply:
I think we should allow people to click active buttons such as "create an epic" during the walkthrough, if they decide they want to go ahead and take that action. On the sample org card, though, it makes less sense to stop the walkthrough when someone clicks that card since the links don't work. Can we can keep the walkthrough open if someone clicks the sample org card?

Reply:
@SondraE Hm, I have a different leaning here. It's a well-established pattern to disable action clicks during a tour/walkthrough, and as a user I think I'd be surprised if I could take actions while the tour is running. I also don't like the idea of treating some actions differently -- I think they should all behave the same. But we have three options:

1. Continue with the default behavior, where clicks are disabled, but stop the tour.
2. Continue disabling clicks, but don't stop the tour if the user clicks somewhere. To stop the tour the user would need to go through all the steps, or click the "x" to close, or type "Esc".
3. Allow all clicks to function normally, but *remove the grey overlay* while the tour is running. This would require a bit of extra logic to disable clicks on the "fake" org, and isn't my favorite option because I think it'll be more confusing for users.

I would definitely lean toward option 1 or 2.

Reply:
@jgerigmeyer Interesting. I had thought when we were looking at options for building the guided tour, we asked Jason/David whether they wanted the user to be able to click to take the actions at any time during the tour (as demoed in https://reactour.js.org/ which, incidentally, appears to be a little broken right now). I thought that they specifically wanted action clicks to be enabled during the guided tour. That said, I too don't love the idea of treating some actions differently. So given this discussion, I would either pick your option 1 or the option to enable clicks and stop the tour.

Reply:
I think we can merge this for now, then ask them when we demo this week.